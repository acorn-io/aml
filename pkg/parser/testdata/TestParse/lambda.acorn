a: lambda x, y: x == y
