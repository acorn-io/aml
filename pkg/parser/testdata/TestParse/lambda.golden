valast: format: 55:18: expected operand, found '==' (and 1 more errors)
